daffodil-cli/src/main/scala/org/apache/daffodil/cli/Main.scala

```diff
@@ -1428,7 +1428,7 @@ class Main(
               forPerformance = true
             )
 
-            val dataSeq: Seq[Either[AnyRef, Array[Byte]]] = files.map { filePath =>
+            val dataSeq: collection.Seq[Either[AnyRef, Array[Byte]]] = files.map { filePath =>
```

Comment:
Why is `collection` needed? Isn't Seq in the global scope?

Reply:
in 2.12, collection.Seq is in the global scope, but in 2.13, it's immutable.Seq resulting in a type mismatch error